tests/fxl-orientation_landscape/OPS/package.opf

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-09-28</dc:date>
+    <dc:description>Reading systems should display or inform the user of the intended orientation of the EPUB.</dc:description>
```

Comment:
Please rewrite to be in the same format as other descriptions: "rendition:orientation is X, so Y".